src/components/Cart/Cart.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import * as actions from "../../store/actions/index"
+import {withRouter} from "react-router-dom"
+import { useSelector, useDispatch } from "react-redux"
+import Navbar from "../Navbar/Navbar"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faPlusSquare, faMinusSquare, faTimes, faCaretRight } from "@fortawesome/free-solid-svg-icons"
+import "./cart.scss"
+function Cart(props) {
+    const dispatch = useDispatch()
+    const cartItems = useSelector(state => state.cartItems)
+    const showCart = useSelector(state => state.showCart)
+const clickHandler=()=>{
+   dispatch(actions.showCart(false))
+    
+}
+    const calculateTotalPrice = () => {
+        const cartItems = useSelector(state => state.cartItems)
+
+        const price = cartItems?.reduce((acc, item) => {
+            return item.count * item.price + acc
+        }, 0)
+        return price
+    }
+    return (
+        <div>
+            {/* <Navbar /> */}
+            <div className="parent" >
+                <div className="cart-heading"><div>My Cart ({cartItems?.length})</div><div onClick={() => dispatch(actions.showCart(!showCart))}><FontAwesomeIcon icon={faTimes} /></div> </div>
```

Comment:
use semantic tags everywhere., why div for heading? again poor indentation , why multiple tags in single line?